patches/openssh/cloudposse/bsd-compatible-realpath.diff

```diff
@@ -0,0 +1,197 @@
+diff --git a/openbsd-compat/openbsd-compat.h b/openbsd-compat/openbsd-compat.h
```

Comment:
This was an `alpine` patch https://git.alpinelinux.org/cgit/aports/tree/main/openssh

Reply:
Move to appropriate vendor folder

Reply:
yeap, but i modified it - now it differs significantly from alpine one